spec/lib/fastly_nsq/rake_task_spec.rb

```diff
@@ -1,34 +1,39 @@
-require 'test_helper'
+require 'spec_helper'
 require 'fastly_nsq/rake_task'
 
-describe MessageQueue::RakeTask do
+RSpec.describe MessageQueue::RakeTask do
   before(:each) do
     Rake::Task['begin_listening'].clear if Rake::Task.task_defined?('begin_listening')
   end
 
-  describe 'defining tasks' do
+  describe 'when defining tasks' do
     it 'creates a begin_listening task' do
       MessageQueue::RakeTask.new
 
       allow_any_instance_of(MessageQueue::RakeTask).to receive(:output) { nil }
-      assert_equal true, Rake::Task.task_defined?(:begin_listening)
+
+      is_defined = Rake::Task.task_defined?(:begin_listening)
+      expect(is_defined).to be(true)
     end
 
     it 'creates a named task' do
       MessageQueue::RakeTask.new(:test_name)
 
       allow_any_instance_of(MessageQueue::RakeTask).to receive(:output) { nil }
-      assert_equal true, Rake::Task.task_defined?(:test_name)
+
+      is_defined = Rake::Task.task_defined?(:test_name)
+      expect(is_defined).to be(true)
     end
   end
 
-  describe 'running tasks' do
+  describe 'when running tasks' do
     it 'runs with inline options defined' do
-      MessageQueue::RakeTask.new(:begin_listening, [:topic, :channel])
+      options = { topic: 'dwarf', channel: 'star' }
+      task = MessageQueue::RakeTask.new(:begin_listening, [:topic, :channel])
 
       dbl = double('go', go: nil)
```

Comment:
Consider renaming this `dbl` here and elsewhere in the file to something more descriptive of what it represents, like `listener = double('MessageQueue::Listener', go: nil)`.